content/tutorials.mdx

```diff
@@ -17,3 +15,9 @@ metaDescription: 'MAIT Tutorials'
 ## UE4
 
 - [Building OpenXR Multi Apps with UE4](./tutorials/ue4_overlay_ext)
+
+## Aardvark
+
+- [Building your first aardvark Gadget](https://aardvarkxr.github.io/aardvark/getting_started/)
```

Comment:
(Title capitalization) Building Your First Aardvark Gadget

Reply:
that tut should be burned with fire but - capitalized! (I appreciate the attention to detail throughout!)